### PR TITLE
Fix the CLI for 204 response from tileset source deletes

### PR DESCRIPTION
**Pull request by flippmoke**
Fix for #52

### CODE REVIEW

File: mapbox_tilesets/scripts/cli.py [hunk 1]
```diff
@@ -384,7 +384,7 @@ def delete_source(username, id, force, token=None):
         mapbox_api, username, id, mapbox_token
     )
     r = requests.delete(url)
-    if r.status_code == 201:
+    if r.status_code == 204:
```
Comment (vartikaz) [inline mapbox_tilesets/scripts/cli.py:387]: where do you get this 204 from?

Comment (flippmoke) [reply mapbox_tilesets/scripts/cli.py:387]: We updated the HTTP status code for delete action this from 201 to 204. This was due to the fact that `201` is for created while `204` is no content.

Comment (vartikaz) [reply mapbox_tilesets/scripts/cli.py:387]: https://github.com/mapbox/api-tilesets/blob/bdd26c2f4be7c972831b14aec297146aae20fede/routes/tilesets-source.js#L98

Comment (vartikaz) [reply mapbox_tilesets/scripts/cli.py:387]: found it. thnx. neat

**Pull request merged**